src/cgr_gwas_qc/workflow/sub_workflows/contamination.smk

```diff
@@ -28,7 +28,16 @@ module thousand_genomes:
         {}
 
 
-use rule pull_b_allele_freq_from_1kg from thousand_genomes
+if cfg.config.user_files.bcf:
+
+    use rule pull_b_allele_freq_from_1kg_bcfinput from thousand_genomes as pull_b_allele_freq_from_1kg with:
+        input:
+            vcf_file=cfg.config.user_files.bcf,
```

Comment:
Should this variable name be `bcf_file`?

Reply:
Renamed in 5009ca51c62d23cb865d0596f697f8456999762e